docs/guides/dynamic-workers.md

```diff
@@ -81,43 +81,47 @@ After each scaling operation, the algorithm enters a cooldown period to prevent
 
 ## Configuration
 
-### Vertical scaler parameters
+### Global Workers Configuration
+
+Add a `workers` property to your runtime configuration (`platformatic.json` or `watt.json`):
 
 | Parameter | Description | Default |
 |-----------|-------------|---------|
-| **maxTotalWorkers** | Maximum total workers across all applications | `os.availableParallelism()` |
-| **maxTotalMemory** | Maximum total memory that can be used by all workers (bytes) | 90% of system total memory |
-| **minWorkers** | Minimum workers for each application | 1 |
-| **maxWorkers** | Maximum workers for each application | `maxTotalWorkers` |
-| **scaleUpELU** | ELU threshold to trigger scaling up (0-1) | 0.8 |
```

Comment:
scaleUpELU and scaleDownELU are still supported. I revert the depreciation a few days ago